tests/api-testing/tests/test_foldersnew.py

```diff
@@ -0,0 +1,142 @@
+def retrieve_new_dashboard(session, base_url, org_id, folder_id):
+    resp_get_dashboard = session.get(
+        f"{base_url}api/v2/{org_id}/folders/dashboards/{folder_id}")
+    print(resp_get_dashboard.content)
+    assert (
+        resp_get_dashboard.status_code == 200
+    ), f"Retrieve these dashboards: Expected 200, but got {resp_get_dashboard.status_code} {resp_get_dashboard.content}"
+
+def delete_new_dashboard(session, base_url, org_id, folder_id):
+    resp_delete_dashboard = session.delete(
+        f"{base_url}api/v2/{org_id}/folders/dashboards/{folder_id}")
+    assert (
+        resp_delete_dashboard.status_code == 200
+    ), f"Deleting these dashboards: Expected 200, but got {resp_delete_dashboard.status_code} {resp_delete_dashboard.content}"
+
+def test_new_all_dashboards(create_session, base_url):
+    """Running an E2E test for all new dashboards."""
+
+    session = create_session
+    org_id = "default"
+
+    resp_new_all_dashboards = session.get(f"{base_url}api/v2/{org_id}/folders/dashboards")
+
+    print(resp_new_all_dashboards.content)
+    assert (
+        resp_new_all_dashboards.status_code == 200
+    ), f"Retrieve all functions list 200, but got {resp_new_all_dashboards.status_code} {resp_new_all_dashboards.content}"
+
+    
+
+
+def test_new_create_dashboards_delete(create_session, base_url):
+    """Running an E2E test for create and delete dashboards folder."""
+
+    session = create_session
+    # Create a dashboards folder
+    org_id = "default"
+    url = base_url
+    payload = {"description": "newfoldernvp", "name": "pytest-automationtests"}
+    resp_create_new_dashboards = session.post(
+        f"{url}api/v2/{org_id}/folders/dashboards", json=payload)
+
+    print(resp_create_new_dashboards.content)
+
+    folder_id = resp_create_new_dashboards.json()["folderId"]
+
+    assert (
+        resp_create_new_dashboards.status_code == 200
+    ), f"Expected 200 dashboards, but got {resp_create_new_dashboards.status_code} {resp_create_new_dashboards.content}"
```

Comment:
This test is creating the "dashboard folder" not the dashboard itself. I'd recommend updating to update the logs to reflect that in order to avoid confusion.